client/src/components/Navbar.js

```diff
@@ -41,41 +43,68 @@ const useStyles = (theme) => ({
 });
 
 function Navbar(props) {
-  const [redirect, setRedirect] = useState(null);
   const [loggedIn, setLoggedIn] = useState(false);
   const [profileImg, setProfileImg] = useState(null);
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [notifications, setNotifications] = useState([]);
+  const [badgeInVisible, setBadgeInVisible] = useState(true);
   const logout = (event) => {
-    event.preventDefault();
-
     axios
       .post('/users/logout')
       .then((response) => {
         if (response.status === 200) {
           setLoggedIn(false);
+          history.push('/');
           localStorage.clear();
-          setRedirect('/');
-          window.location.reload();
         }
       })
       .catch((error) => {
         console.log(error);
       });
   };
 
+  const handleNotificationClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleNotificationClose = () => {
+    setAnchorEl(null);
+  };
+
   useEffect(() => {
     const token = localStorage.getItem('loginToken');
     if (token !== null) {
-      setRedirect('/');
       axios.get(`/profile/ref/${props.userID}`).then(({ data }) => {
-        setProfileImg(`https://team-kiwi.s3.ca-central-1.amazonaws.com/${data.profileImg}`);
+        setProfileImg(`${process.env.REACT_APP_S3_IMAGE_URL + data.profileImg}`);
+
+        // If current user is on Sitter//
+        // Get available requests by using current sitter ID
+        axios.get(`/request/getSitterRequest/5f12210308f7260dab867b09`).then(({ data }) => {
```

Comment:
There is static profile_id

Reply:
yeah, I did hard coded it temporarily, because don't know the current user status yet